diff --git a/src/DLP.cpp b/src/DLP.cpp
index 7016598..77ade15 100644
--- a/src/DLP.cpp
+++ b/src/DLP.cpp
@@ -101,12 +101,15 @@ bool DLP::DLPCheck(CUpDownClient* c){
 	if (tmp != NULL) {
 		ret = tmp;
 		wxString wxInfo;
-		wxInfo.Printf(wxT("[%s] %s"), ret.c_str(), c->GetClientFullInfo().c_str());
+		wxInfo.Printf(wxT("[%s] %s [%s]"), ret.c_str(), c->GetClientFullInfo().c_str(), uhash.c_str());
 		c->Ban();
 		theApp->AddDLPMessageLine(wxInfo);
 		return true;
 	}
 
+	wxString wxInfo;
+	wxInfo.Printf(wxT("[Record IP] %s [%s]"), c->GetClientFullInfo().c_str(), uhash.c_str());
+	theApp->AddDLPMessageLine(wxInfo);
 	return false;
 
 }
